master/internal/provisioner/gcp.go

```diff
@@ -60,6 +62,19 @@ func newGCPCluster(config *Config) (*gcpCluster, error) {
 
 	startupScriptBase64 := base64.StdEncoding.EncodeToString([]byte(config.StartupScript))
 	containerScriptBase64 := base64.StdEncoding.EncodeToString([]byte(config.ContainerStartupScript))
+
+	var certBytes []byte
```

Comment:
We can actually move this part shared in `aws.go` and `gcp.go` to `provisioner.go` `func New()`.

Reply:
There's other preexisting shared stuff between the two, so I think I'll follow up with another PR to deal with all of it together.